CHANGELOG.develop

```diff
@@ -435,6 +446,7 @@ Benner and Paweł Siudak):
   - The new =spacemacs-modeline= layer configures packages for the mode line.
   - The new =spacemacs-project= layer configures =projectile=.
   - Merge =spacemacs-ui= and =spacemacs-ui-visual= into layer =spacemacs-visual=
+  - Remove =spacemacs-environment= from =spacemacs-bootstrap= layer
```

Comment:
The spacemacs-environment layer was not in the last release, so this entry can probably be omitted.